### PR TITLE
LWDEV-10406 One thread for all throttling loggers

### PR DESCRIPTION
**Pull request by barsukovskii**
Currently each ThrottlingLogger instance creates a separate thread to clear their logged errors cache.
Changed kotlin-utils lib - added single thread with one common logged errors cache for all loggers.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -32,7 +32,7 @@ repositories {
 }
 
 ext {
-    kotlin_utils_version = '0.0.8-SNAPSHOT'
+    kotlin_utils_version = '0.0.9-SNAPSHOT'
```
Comment (rocketmannot) [inline build.gradle]: why do we use SNAPSHOT - version?

Comment (barsukovskii) [reply build.gradle]: I can't publish not SNAPSHOT artefact to https://oss.sonatype.org/content/repositories/snapshots/. Perhaps it is connected with rights of my account I created for this library once )

Comment (rocketmannot) [reply build.gradle]: suggest to use the lates version

Comment (barsukovskii) [reply build.gradle]: the latest one is not compatible with log4j in this version

**Pull request merged**